tests/rest/test_job.py

```diff
@@ -264,6 +265,124 @@ def test_get_job_logs(session040, requests_mock):
     assert log_entries[0].message == "error processing batch job"
 
 
+# TODO: do we keep testing on sesssion040, or should we switch to con100?
+def test_get_job_logs_returns_only_error_loglevel_by_default(session040, requests_mock):
+    requests_mock.get(
+        API_URL + "/jobs/f00ba5/logs",
+        json={
+            "logs": [
+                {
+                    "id": "123abc",
+                    "level": "error",
+                    "message": "error processing batch job",
+                },
+                {
+                    "id": "234abc",
+                    "level": "debug",
+                    "message": "Some debug info we want to filter out",
+                },
+                {
+                    "id": "345abc",
+                    "level": "info",
+                    "message": "Some general info we want to filter out",
+                },
+                {
+                    "id": "345abc",
+                    "level": "warning",
+                    "message": "Some warning we want to filter out",
+                },
+            ]
+        },
+    )
+
+    log_entries = session040.job("f00ba5").logs()
+
+    assert len(log_entries) == 1
+    assert log_entries[0].message == "error processing batch job"
+
+
+@pytest.mark.parametrize(
+    "log_level,exp_num_messages",
```

Comment:
FYI: you don't have to comma-serialize multiple parameters  as single string, you can just provide a list `["log_level", "exp_num_messages"]`

Reply:
Done.